viewer/Viewer.cpp

```diff
@@ -106,26 +107,55 @@ namespace Enki
 		yaw(yaw),
 		pitch(pitch)
 	{
-		
+		update(false);
 	}
 	
+	void ViewerWidget::CameraPose::update(bool trackballMode, QVector3D targetPosition, float zNear)
+	{
+		forward = QVector3D( cos(yaw)*cos(pitch), sin(yaw)*cos(pitch), sin(pitch) );
+		left = QVector3D::crossProduct(QVector3D(0,0,1), forward).normalized();
+		up = QVector3D::crossProduct(forward, left).normalized();
+
+		if(trackballMode)
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.